indexer/Worker.cc

```diff
@@ -689,6 +690,24 @@ class IndexerAstVisitor : public clang::RecursiveASTVisitor<IndexerAstVisitor> {
     return true;
   }
 
+#define VISIT_TYPE_LOC(TypeName)                                    \
+  bool Visit##TypeName##TypeLoc(clang::TypeName##TypeLoc typeLoc) { \
+    this->tuIndexer.save##TypeName##TypeLoc(typeLoc);               \
+    return true;                                                    \
+  }
+  FOR_EACH_TYPE_TO_BE_INDEXED(VISIT_TYPE_LOC)
+#undef VISIT_TYPE_LOC
+
+  bool TraverseNestedNameSpecifierLoc(
+      const clang::NestedNameSpecifierLoc nestedNameSpecifierLoc) {
+    // Unlike many other entities, there is no corresponding Visit* method in
```

Comment:
Feel like this comment is more appropriate on the declaration and not inside it.

What do you think about marking overridden methods with override for the extra checking that entails.

Reply:
Moved the comment.

I'll add override markers in a separate PR, that's a good point, I somehow just forgot about adding those.

Reply:
Ah right, this is using the CRTP pattern, so even though these are technically overrides, since they're not virtual methods, they can't be marked override.

I think we'd have gotten a warning on missing the override otherwise since all warnings are enabled.